src/west/app/project.py

```diff
@@ -932,18 +920,17 @@ def update_submodules(self, project):
             return
 
         submodules = project.submodules
-        call_location = os.path.join(self.topdir, project.path)
         submodules_update_strategy = ('--rebase' if self.args.rebase
                                       else '--checkout')
         # For the list type, update given list of submodules.
         if isinstance(submodules, list):
             for submodule in submodules:
-                project.git(['-C', call_location, 'submodule', 'update',
+                project.git(['submodule', 'update',
```

Comment:
Just to clarify `call_location` was used to allow calling git command from the project location and update submodules there, so submodules paths could be relative to this place. Did you remove it, because by default command is called from the project directory, or submodules paths should be absolute?

Reply:
> . Did you remove it, because by default command is called from the project directory, or submodules paths should be absolute?

The first one. Project.git() runs in the project's abspath by default:

https://docs.zephyrproject.org/latest/guides/west/west-apis.html#west.manifest.Project.git